change/@fluentui-react-file-type-icons-2021-02-18-14-27-11-expose-file-type-icons-cherrypick-v8.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "prerelease",
+  "comment": "cherrypick filetypeicons as html string to master branch",
```

Probably better to reuse the original change message
```suggestion
  "comment": "Icons can now be requested using the getFileTypeIconAsHTMLString API, for compatibility with non-React environments.",
```